src/core/server/app/handlers/api/story/count.ts

```diff
@@ -35,6 +41,44 @@ interface StoryCountJSONPQuery {
   ref: string;
 }
 
+function getCountHTML(
+  tenant: Readonly<Tenant>,
+  storyMode: GQLSTORY_MODE | undefined | null,
+  i18n: I18n,
+  count: number
+) {
+  // Use translated string.
+  const bundle = i18n.getBundle(tenant.locale);
+
+  let html = "";
+
+  if (storyMode === GQLSTORY_MODE.RATINGS_AND_REVIEWS) {
+    html = translate(
```

Comment:
What do you think about doing something like:
```
const text =
    storyMode === GQLSTORY_MODE.RATINGS_AND_REVIEWS ? "Ratings" : "Comments";
  const localizationID =
    storyMode === GQLSTORY_MODE.RATINGS_AND_REVIEWS
      ? "comment-count-ratings-and-review"
      : "comment-count";
  const html = translate(
    bundle,
    `<span class="${NUMBER_CLASS_NAME}">${count}</span> <span class="${TEXT_CLASS_NAME}">${text}</span>`,
    `${localizationID}`,
    {
      number: count,
      numberClass: NUMBER_CLASS_NAME,
      textClass: TEXT_CLASS_NAME,
    }
  );
```

This would have less repetition and I think would be clearer the difference between the two.

Reply:
The reason I'm doing this with explicit if/else is that we're likely going to see a need to do this for QA mode as well. Also, I need to do yet another mode for a special SBN count mode. So I want to get it ready for what will be 4 flows instead of just 2.

That being said, I like your idea to mitigate duplication, but in this case, to keep the rendering isolated to what will be separate flows, I'm going this route with a bit of duplication.